LowLevelTestCases/TestCasesEDIT.md

```diff
@@ -0,0 +1,129 @@
+# Niskopoziomowe Przypadki Testowe dla Edytora Tekstowego "Cosmic"
+
+## 1. TC-EDIT-01: Weryfikacja możliwości otwarcia pliku tekstowego
+**Cel:** Sprawdzić, czy użytkownik może otworzyć dowolny plik tekstowy.
+
+**Warunki wstępne:**
+- Oprogramowanie "Cosmic" musi być zainstalowane na systemie Linux.
```

Comment:
Jest to warunek niezbędny do rozpoczęcia testowania, według planu testów, redundantnym jest powtarzanie go tutaj